src/main.py

```diff
@@ -61,9 +62,13 @@ def remove_games(self, games):
 
     def get_games(self):
         return self.record["games"]
-
+    
+    def get_blacklist(self):
+        return self.record["blacklist"]
 
 class WhatshouldWePlayBot(discord.Client):
+    _ignore_list = [959263650701508638, 961433803484712960]
```

Comment:
since `_ignore_list` list never changes, it would be more appropriate as a "constant" (which python does not trully have) which is to make it upper case. and I would probably add the word "member" to the word. I had to dig around for 30 seconds or so to figure out what this even is. If the project gets bigger, that will just take more time.

```suggestion
    _MEMBER_IGNORE_LIST = [959263650701508638, 961433803484712960]
```

I also think you should at least have a comment regarding what these two random numbers are.